docs/install/win/activating-a-validator.md

```diff
@@ -54,54 +52,46 @@ This command will output a `Raw Transaction Data` block:
 
 **Method 2:** Users may also choose to submit the required 32 GöETH along with the data directly to the current deposit contract ****\(found [here](https://prylabs.net/contract)\). To enable the hex data field on the Send page in Metamask, click your wallets avatar &gt; Settings &gt; Advanced &gt; toggle 'Show hex data'.
 
-## Starting up the beacon node
-
-**NOTICE:** If you have already started and syncronised your beacon node by following the [Getting Started](./#connecting-to-the-testnet-running-a-beacon-node) guide section on the topic, this portion can be skipped.
+## Step 4a: Starting up the beacon node
```

Comment:
Thought on this numbering scheme @rauljordan @prestonvanloon? If we go with this, it will need to replicated throughout each installation and validator guide for consistency

Reply:
I did this to align with the numbering scheme on https://prylabs.net/participate  I'm already in the process of replicating for consitency across all OS's

Reply:
Aligned for all OS's